app/views/static_pages/board.html.erb

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html >
+<html>
+<style type="text/css">
+ 	.board {
+ 		width: 640px;
+    	height: 655px;
+    	border: 3px solid #333;
+    	margin: 10px auto 10px auto;
+
+ 	}
+ 	body {
+ 		 padding-top: 30px; 
+ 		 padding-bottom: 30px;
+ 		
+ 	}
+	td {
+ 		empty-cells: show;
+ 		height: 80px;
+ 		width: 80px;
+ 		vertical-align:middle;
+ 	}
+ 	table.one {
+ 		border-collapse: collapse;
+ 	}
+ 	td.black {
+ 		background-color: #000608;
+ 	}
+ 	</style>
+	<body>
+	<div class="board">
+ 		<table class="one">
```

Comment:
We've got a lot of repetition here. Maybe a lil somethin somethin with some loops? Take a look at Ruby Integeter `.upto` and `.downto` methods